app/script/assets/Asset.js

```diff
@@ -0,0 +1,62 @@
+//
+// Wire
```

Comment:
We use the copyright template with asterisks: https://github.com/wireapp/wire-open-source/blob/master/preamble/preamble.js